torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -98,6 +98,9 @@ AOTI_TORCH_EXPORT AOTI_TORCH_NOINLINE int32_t aoti_torch_dtype_int16();
 AOTI_TORCH_EXPORT AOTI_TORCH_NOINLINE int32_t aoti_torch_dtype_int32();
 AOTI_TORCH_EXPORT AOTI_TORCH_NOINLINE int32_t aoti_torch_dtype_int64();
 
+AOTI_TORCH_EXPORT AOTI_TORCH_NOINLINE int64_t
```

Comment:
@ezyang , I really don't want to return AOTITorchError for this one. I guess theoretically we can hit a div_by_zero exception, but I wonder if it is really worth to do that.

Reply:
Maybe a better way is to duplicate https://github.com/pytorch/pytorch/blob/1cc0921eb62089392595264610cfa863d42fe9bc/aten/src/ATen/native/BinaryOps.h#L97 (without using c10 functions in it) in model.so for simple utility function like this?

Reply:
uhhhh why are you sending this across the ABI boundary, why not just have an implementation in model.so

Reply:
yeah what you suggested